pkg/arvo/sys/zuse.hoon

```diff
@@ -5474,6 +5475,16 @@
       |=  a/^time
       =-  (numb (div (mul - 1.000) ~s1))
       (add (div ~s1 2.000) (sub a ~1970.1.1))
+    ::                                                  ::  ++path:enjs:format
+    ++  path                                            ::  string from path
+      |=  a=^path
+      ^-  json
+      [%s (spat a)]
+    ::                                                  ::  ++tank:enjs:format
+    ++  tank                                            ::  tank as string arr
+      |=  a=^tank
+      ^-  json
+      [%a (turn (wash [0 80] a) tape)]
```

Comment:
Looks like +wash produces a $wall already.  We shouldn't have to coerce to $tape for each line in the $wall, which will have O(n) performance.

Reply:
It's calling `+tape:enjs:format`, which will render each line to a JSON string inside thiis array.

Reply:
oh ok